### DISCUSSION TITLE
update docs with code more inline with nuxt builtin features

### DISCUSSION THREAD

**Discussion by sandros94** [Intevel/nuxt-directus#149] [Ideas]
So, this idea came to mind while discussing #148 (btw you could move the Volta's status from `canceled` to `done`, since now I marked it as completed).

Would you like the idea that I go and update all the docs code to use function like `useAsyncData` like discussed? There are two reasons why I'm considering this:

- When I started coding js/ts only a few months ago it was hard for me to understand what I was doing, part of the reason being ts interfaces and the fact that I didn't have any connections between nuxt-directus docs and nuxt docs.
- Easier integration of additional functionalities: as a beginner one of the things I've struggled a bit (before Daniel pointed out the `refresh` value) was integrating a way to edit the parameters of the fetch and trigger a refresh. While it was working, I wasn't sure I was doing it in the right way (part of it because of the point above). But yesterday once I was pointed to the right directions on "what docs I should look at" everything seemed so simple and fast to integrate.

The reason why I'm opening a discussion instead of straight creating a PR is just to make sure that you weren't already doing something similar, and I would have only created confusion in the process.

**Comment by casualmatt** [comment]
We can add an example of "how to use useAsyncData" but the documentation needs to be easy to read even for not expert developers, so I would prefer to keep it as it is.

**Comment by casualmatt** [comment]
Example added in #153
